config/dev.json

```diff
@@ -1,6 +1,7 @@
 {
   "agoraUrlRoot": "https://agora.dsde-dev.broadinstitute.org",
   "bardRoot": "https://terra-bard-dev.appspot.com",
+  "billingProjectManagerUrlRoot": "https://bpm.dsde-dev.broadinstitute.org",
```

Comment:
Currently dev is the only one that exists, but everything then prod should be available shortly. And Azure support is not enabled in prod, so we won't attempt to hit the prod BPM service.